package.json

```diff
@@ -10,10 +10,10 @@
   "engines": {
     "node": "8.6.0"
```

Comment:
While we're in here should we take this whole piece out, too?  I think we had talked about it before and if anything we should make sure we're running/using the latest Node.js LTS version (8.9.4).

Reply:
You're right. Thanks!